### PR TITLE
Some config.sample.php fixes

### PR DESCRIPTION
**Pull request by mmattel**
## Description

Issue 1: Fixed a mistake in using the app-name instead the app-id (thanks to @thommierother)
Issue 2: Improved the text for hashing cost, minor change

## Related Issue

https://github.com/owncloud/core/pull/31066#issuecomment-381091162

## Motivation and Context

Fix a mistake and improve a text

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -1189,13 +1191,13 @@
...
- * because themes are treated as apps.
- * The following example allows theme-1 and theme-2 to have no signature.
+ * because themes are treated as apps. The app is identified with it´s app-id. 
+ * The following example allows app-1 and theme-2 to have no signature.
  */
 'integrity.ignore.missing.app.signature' =>
 	array(
-		'theme-1',
-		'theme-2',
+		0 => 'app-id of app-1',
```
Comment (PVince81) [inline config/config.sample.php]: you don't need to manually add the "0 =>" part as it's already an index-based array.

however PHP will write those out when re-saving config.php

**Comment by mmattel** [comment]
@PVince81: changed

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by mmattel** [comment]
Thanks @PVince81 
Can you look on the backport which just passed checks.

**Pull request merged**